_episodes/01-introduction.md

```diff
@@ -15,7 +15,7 @@ shape of the objects in an image)."
 - "Recognize colorimetric problems (those dealing with the analysis of the
 color or the objects in an image)."
 keypoints:
-- "Simple Python and OpenCV techniques can be used to solve genuine 
+- "Simple Python and skimage techniques can be used to solve genuine 
```

Comment:
I would introduce the "full name" first. I know there is this link below, that also has the full name, but since this is written to beginners, it doesn't hurt.

```suggestion
- "Simple Python and skimage (scikit-image) techniques can be used to solve genuine 
```

Reply:
Good point.